### DISCUSSION TITLE
Week long schedule with start, stop and day.... Sunday, Monday etc

### DISCUSSION THREAD

**Discussion by ChuckNull** [gopro/labs#1182] [General]
Would be helpful if the schedule could have day of the week along with start and stop times so if one was recording a construction site Mon -Fri

**Comment by dnewman-gpsw** [comment]
The day of the week is stored in the Labs variable ```t:W```. Try scanning ```"Day $t:W"```  

So you would test t:W in your script to branch for day of the week.

```
!09:00N <- Wake a 9am daily
=Dt:W   <- Load the day of the week in variable D
>D6!R   <- If D (day of the week) is >= 6 (Saturday) sleep again.
!S          <- start your week day capture
etc.
```

So the QR Command might look like this:
```!09:00N=Dt:W>D6!RmT!S!17:00E!R```

A timelapse capture from 9am to 5pm on weekdays.

The command language is documented here: https://gopro.github.io/labs/control/tech/
